.github/workflows/format.yml

```diff
@@ -0,0 +1,31 @@
+name: Format Files
+on:
+  pull_request:
+    branches: [master]
+permissions:
+      # Give the default GITHUB_TOKEN write permission to commit and push the
+      # added or changed files to the repository.
+      contents: write
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '18.x'
+      - name: Install Prettier
+        run: npm install -g prettier@2.8.0
+      - name: Run Formatter
+        run: yarn format
+      - name: Set Author Details
+        run: |
+            git config --global user.email "rlingineni@pinterest.com"
```

Comment:
You can use the same account we use for releases, [pinterest-gestalt](https://github.com/pinterest-gestalt)

I would set the email as `pinterest-gestalt@gmail.com` and name as "Pinterest-Gestalt"